src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	String getPrintableString();
```

Comment:
Good job on not having any access modifiers (`public`, `private`, or `protected`). Access control is all about hiding implementation details. Since an interface has no implementation to hide, there is no need to have any access modifiers.